projects/fsastorefrontlib/src/cms-components/my-account/claim/claim-details/claim-details.component.html

```diff
@@ -1,25 +1,17 @@
 <ng-container *ngIf="claim$ | async as claim">
   <div class="overview-section mb-5">
-    <span class="image-wrapper">
-      <cx-media [container]="claim.incidentType?.icon"></cx-media>
-    </span>
-    <h4>
-      <span class="overview-section-title">
-        {{ claim.insurancePolicy?.categoryData?.name }}
-        {{ 'claim.claim' | cxTranslate }}
-        <br />
-      </span>
-      <span class="overview-section-value border-bottom border-color-3">
-        {{ claim.claimNumber }}
-      </span>
-    </h4>
     <a
-      class="overview-section-link"
+      class="h4 overview-section-link"
```

Comment:
In consultation with designer, removed excess code, to sync the styling with other pages